### PR TITLE
Update helm chart versions and kubeflow versions

### PR DESCRIPTION
**Pull request by jsitu777**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**
* update kubeflow versions from v1.6.1-rc.0 to v1.6.1
* update helm chart app versions based on component versions https://github.com/kubeflow/manifests/tree/v1.6.1#kubeflow-components-versions
* increment chart version in helm charts and terraform locals.tf for pipeline and istio

**Testing:**
- [ ] Unit tests pass
- [ ] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: charts/apps/volumes-web-app/Chart.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
```
Comment (surajkota) [inline charts/apps/volumes-web-app/Chart.yaml:21]: bump chart versions as well for these apps as well?

File: charts/common/aws-secrets-manager/rds-only/Chart.yaml [hunk 2]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "v3.3.5"
```
Comment (surajkota) [inline charts/common/aws-secrets-manager/rds-only/Chart.yaml:24]: why 3.3.5?

I think we can stick to 1.6.1 for this as well

File: charts/common/kubeflow-issuer/Chart.yaml [hunk 3]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "v1.5.0"
```
Comment (surajkota) [inline charts/common/kubeflow-issuer/Chart.yaml:24]: 1.5?

File: charts/common/oidc-authservice/Chart.yaml [hunk 4]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "v1.3"
```
Comment (surajkota) [inline charts/common/oidc-authservice/Chart.yaml:24]: reference to 1.3?

File: charts/common/kubeflow-istio-resources/Chart.yaml [hunk 5]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "1.14.1"
```
Comment (surajkota) [inline charts/common/kubeflow-istio-resources/Chart.yaml:24]: I think these are specific to Kubeflow so v1.6.1 is more appropriate, same for cluster-local-gateway

**Review by surajkota — CHANGES_REQUESTED** [review]

File: charts/common/kubeflow-issuer/Chart.yaml (hunk 3 above)
Comment (jsitu777) [reply charts/common/kubeflow-issuer/Chart.yaml:24]: since cert-manager is v1.5.0

File: charts/common/oidc-authservice/Chart.yaml (hunk 4 above)
Comment (jsitu777) [reply charts/common/oidc-authservice/Chart.yaml:24]: https://github.com/arrikto/oidc-authservice
Then it points to "This is a rewrite of the [ajmyyra/ambassador-auth-oidc](https://github.com/ajmyyra/ambassador-auth-oidc) project."
https://github.com/ajmyyra/ambassador-auth-oidc
https://github.com/ajmyyra/ambassador-auth-oidc/commit/a07b7bdf1e96aebf4d5b661f97b3fd35bda7f065

File: charts/common/aws-secrets-manager/rds-only/Chart.yaml (hunk 2 above)
Comment (jsitu777) [reply charts/common/aws-secrets-manager/rds-only/Chart.yaml:24]: https://github.com/awslabs/kubeflow-manifests/blob/54834fd03d0e241c7908b4a5e2d22e9a15267dbb/awsconfigs/common/aws-secrets-manager/base/deployment.yaml#L19
I got the version from the tag here:
https://gallery.ecr.aws/xray/aws-xray-daemon

File: charts/common/kubeflow-istio-resources/Chart.yaml (hunk 5 above)
Comment (jsitu777) [reply charts/common/kubeflow-istio-resources/Chart.yaml:24]: ok

File: charts/common/aws-secrets-manager/rds-only/Chart.yaml (hunk 2 above)
Comment (surajkota) [reply charts/common/aws-secrets-manager/rds-only/Chart.yaml:24]: I see, thats a dummy image we use for this component, its used to just create a pod for mounting purposes. we can do 1.0.0 here

File: charts/apps/volumes-web-app/Chart.yaml (hunk 1 above)
Comment (jsitu777) [reply charts/apps/volumes-web-app/Chart.yaml:21]: bumped chart versions for admission weboook, jupyter, notebook controller

File: charts/common/kubeflow-issuer/Chart.yaml (hunk 3 above)
Comment (surajkota) [reply charts/common/kubeflow-issuer/Chart.yaml:24]: got it, this is kubeflow specific so can do 1.6.1 here

File: charts/common/oidc-authservice/Chart.yaml (hunk 4 above)
Comment (surajkota) [reply charts/common/oidc-authservice/Chart.yaml:24]: strange, there are no tags or releases on this repo. for this, lets make it same as chart version

Comment (jsitu777) [reply charts/common/oidc-authservice/Chart.yaml]: done

File: charts/common/kubeflow-issuer/Chart.yaml (hunk 3 above)
Comment (jsitu777) [reply charts/common/kubeflow-issuer/Chart.yaml]: done

File: charts/apps/central-dashboard/Chart.yaml [hunk 6]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
```
Comment (surajkota) [inline charts/apps/central-dashboard/Chart.yaml:21]: update chart version

File: charts/apps/models-web-app/Chart.yaml [hunk 7]
```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.1.1
```
Comment (surajkota) [inline charts/apps/models-web-app/Chart.yaml:18]: this did not change in 1.6.1.

File: charts/apps/notebook-controller/Chart.yaml [hunk 8]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
```
Comment (surajkota) [inline charts/apps/notebook-controller/Chart.yaml:21]: update chart version

File: charts/apps/profiles-and-kfam/Chart.yaml [hunk 9]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
```
Comment (surajkota) [inline charts/apps/profiles-and-kfam/Chart.yaml:21]: update chart version

File: charts/apps/tensorboard-controller/Chart.yaml [hunk 10]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
```
Comment (surajkota) [inline charts/apps/tensorboard-controller/Chart.yaml:21]: update chart version

File: charts/apps/tensorboards-web-app/Chart.yaml [hunk 11]
```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
```
Comment (surajkota) [inline charts/apps/tensorboards-web-app/Chart.yaml:21]: update chart version

File: charts/apps/volumes-web-app/Chart.yaml (hunk 1 above)
Comment (surajkota) [inline charts/apps/volumes-web-app/Chart.yaml:21]: update chart version

**Review by surajkota — COMMENTED** [review]
also update corresponding chart versions in terraform

**Review by surajkota — APPROVED** [review]

**Pull request merged**